PSO-Application.md

```diff
@@ -0,0 +1,114 @@
+# PlanetaryPy
+
+## Introduction
+
+PlanetaryPy will be a community effort to develop a core package
+for planetary science in Python and foster interoperability between
+Python planetary science packages.
+
+In order to facilitate that goal, the [PlanetaryPy
+TC](https://github.com/planetarypy/TC) is applying to become a
+Top-Level Project in the Planetary Software Organization.
+
+
+## History & Outlook
+
+The concept of PlanetaryPy has existed since at least 2015.  Austin
+Godber, and other developers created the planetarypy GitHub
+organization in 2015, and began maintaining several repositories
+there.  There was no governance model, and no central "planetarypy"
+package.  Many of these original developers moved on from planetary
+sciences, and active maintenance of these repositories waned.
+
+In 2019, there was renewed interest by a new group of individuals
+(now the members of the PlanetaryPy TC) to create something for
+planetary sciences that was akin to the Astropy project for astronomy.
+
+At the Planetary Data Workshop in June, 2019, we found that there
+was a real interest in the Planetary Sciences technical community
+for an active planetary Python resource. Many people pointed to the
+astropy project as a possible analog or goal.  The code currently
+within the GitHub PlanetaryPy organization is a great starting
+point, and the naming is right.
+
+The goal is to take that existing codebase, plus a solid governance
+model taken from the Planetary Software Organization, and eventually
+work to build something that is as robust as the Astropy Project.
+We recognize that there is a large gulf between where we are now,
+and something as comprehensive as Astropy, and the intent is to try
+and slowly and methodically build.
+
```

Comment:
is that sentence's grammar complete and okay? Feels broken off?

Reply:
I'll admit its a little grammatically artistic.  Suggest a revision.

Reply:
I was wondering if we even need to evaluate our current status compared to astropy. Isn't it sufficient to pronounce it as our target?

Reply:
Good point.  Take a look at the new version, see if you like that better.